tasks/copy_to_glacier/test/unit_tests/test_copy_to_glacier.py

```diff
@@ -127,7 +127,7 @@ def test_handler_happy_path(self, mock_task: MagicMock):
         handler_input_event = {
             "payload": {"granules": granules},
             "task_config": {
-                CONFIG_EXCLUDE_FILE_TYPES_KEY: [".png"],
+                CONFIG_ORCA_EXCLUDED_FILE_EXTENSIONS: [".png"],
```

Comment:
Possibly change to `CONFIG_ORCA_EXCLUDED_FILE_EXTENSIONS_KEY` to match naming convention

Reply:
updated